src/components/Auth/LandingFooter.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, memo } from 'react';
+
+import { useCurrentCriteriaContext } from 'components/CurrentCriteriaProvider';
+import mitre from 'assets/mitre.png';
+import mcode from 'assets/mcode.png';
+import useStyles from './styles';
+
+const LandingFooter: FC = () => {
+  const styles = useStyles();
+  const { resetCurrentCriteria } = useCurrentCriteriaContext();
+
+  return (
+    <footer className={styles.footer}>
+      <a
```

Comment:
Clicking on either of these links gives 
`TypeError: resetCurrentCriteria is not a function`
and then does not load the website in the new tab. I think its because the routing happens in `index.js` so the App is never loaded (which is where the currentCriteriaContext is stored).

In fact, why does this component even need to resetCurrentCriteria()?

Reply:
Oops again, fixed!